PluginCore/PluginCore/Controls/CompletionList.cs

```diff
@@ -364,7 +364,7 @@ static private void CLDrawListItem(Object sender, System.Windows.Forms.DrawItemE
                 g.InterpolationMode = System.Drawing.Drawing2D.InterpolationMode.HighQualityBicubic;
                 g.DrawImage(item.Icon, 1, e.Bounds.Top + ((e.Bounds.Height - newHeight) / 2), newHeight, newHeight);
                 int p = item.Label.LastIndexOf('.');
-                if (p > 0 && !selected)
+                if (p > 0 && !selected && !(item is ICompletionListSpecialItem))
```

Comment:
Added a new interface instead of adding a new property to the existing interface, since that will cause all other classes implementing it to break.